AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -912,7 +912,9 @@ - (ASCellNode *)nodeAtIndexPath:(NSIndexPath *)indexPath
       context = completedNodesSection[row];
     }
   }
-  
+  if (_inverted) {
+    context.node.transform = CATransform3DMakeRotation(M_PI, 0, 0, 1.0);
```

Comment:
Yeah that makes sense, I had opted to avoid this initially so that I didn't have to duplicate code, but for the sake of two extra lines of code, it would feel nice not to have the ASDataController be involved.